deltatech_stock_inventory/models/stock_inventory.py

```diff
@@ -194,6 +206,58 @@ def post_inventory(self):
         # This is a normal behavior
         # as quants cannot be reuse from inventory location (users can still manually move the products before/after
         # the inventory if they want).
+
+        # archive old SVL's and write new svl if checked
+        # TODO: test and correct with storage stock sheet
+        if self.archive_svl:
+            for line in self.line_ids:
+                # archive old svls
+                svls = self.env["stock.valuation.layer"].search([("product_id", "=", line.product_id.id)])
```

Comment:
trebuie sa arhivezi mai intai SVL-urile de la toate produsele in inventar si apoi in alt for te apuci sa faci altele SVL-uri. Daca un produs ie in doua locatii la a doau liniie arhivezi si SVl facut ainterior.